app/views/simplified-account/settings/email-notifications/edit-email-collection-mode.njk

```diff
@@ -0,0 +1,69 @@
+{% extends "../settings-layout.njk" %}
+
+{% block settingsPageTitle %}
+  Email notifications - {{currentService.name}} {{currentGatewayAccount.full_type}} - GOV.UK Pay
+{% endblock %}
+
+{% block settingsContent %}
+
+    {{ govukBackLink({
+      text: "Back",
+      href: backLink
+    }) }}
+
+    <form id="collection-email-form" method="post" novalidate>
+      <input id="csrf" name="csrfToken" type="hidden" value="{{csrf}}"/>
+
+      {{
+        govukRadios({
+          idPrefix: 'email-collection-mode',
+          name: 'emailCollectionMode',
+          fieldset: {
+            legend: {
+              text: 'Do you want to ask users for an email address on the card payment page?',
+              isPageHeading: true,
+              classes: 'govuk-fieldset__legend--l'
+            }
+          },
+          items: [
+            {
+                value: emailCollectionModes.mandatory,
+                text: 'Yes – as a mandatory field',
+                checked: emailCollectionMode === 'MANDATORY',
+                hint: {
+                  text: 'Users must enter an email address to complete a payment – they can receive notifications'
+                },
+                disabled: isReadyOnly
+            },
+            {
+                value: emailCollectionModes.optional,
+                text: 'Yes – as an optional field',
+                checked: emailCollectionMode === 'OPTIONAL',
+                hint: {
+                  text: 'Users can choose to enter an email address – if they do, they can receive notifications'
+                },
+                disabled: isReadyOnly
+            },
+            {
+                value: emailCollectionModes.no,
+                text: 'No',
+                checked: emailCollectionMode === 'OFF',
+                hint: {
+                  text: 'Users do not have the option to enter an email address – they will not receive notifications'
+                },
+                disabled: isReadyOnly
+            }
+          ]
+        })
+      }}
+      {% if not isReadyOnly %}
```

Comment:
```suggestion
      {% if not isReadOnly %}
```

Reply:
This was due to bad merge and should have been here 😩